pkg/mdformatter/linktransformer/link.go

```diff
@@ -180,6 +190,20 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 	v.c.OnError(func(response *colly.Response, err error) {
 		v.rMu.Lock()
 		defer v.rMu.Unlock()
+		// Colly stores requests in cache, even when they have errors.
+		// So need to delete requests with errors from cache database.
```

Comment:
OMG this is quite annoying, can we add an issue on Colly to propose a different API?

Reply:
Yes, I plan to add a few issues. 🙂